python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/management-api-gnmi.j2

```diff
@@ -32,6 +32,9 @@ management api gnmi
 {%                     if transport.notification_timestamp is arista.avd.defined %}
       notification timestamp {{ transport.notification_timestamp }}
 {%                     endif %}
+{%                     if transport.authorization_requests is arista.avd.defined(true) %}
```

Comment:
Order is wrong, this should be between ip access group and notification timestamp option, checked on device and this is the correct order:

management api gnmi
    transport grpc dogeface
+      shutdown
+      ssl profile doge
+      port 2345
+      vrf FACE
+      ip access-group DOGE
+      qos dscp 46
+      authorization requests
+      accounting requests
+      certificate username authentication
+      notification timestamp send-time
+      listen-addresses 12.3.4.5

Reply:
done